Gemfile

```diff
@@ -90,6 +90,7 @@ group :test do
   gem "rspec-rails", "~> 5.0"
   gem "selenium-webdriver", "~> 3.142"
   gem "webdrivers", "~> 4.6.0"
+  gem "rails-controller-testing", "~> 0.0.3"
```

Comment:
Using system tests instead of controller tests will allow us to avoid installing additional dependencies for controller testing.